tests/test_janitor.py

```
@@ -21,24 +21,32 @@
+    with pytest.warns(DeprecationWarning, match="version argument is deprecated"):
+        janitor = DatabaseJanitor(user="user", host="host", port="1234", dbname="database_name", version=version)
     assert janitor.version == VERSION
 
 
+def test_version_is_optional() -> None:
+    """Test that janitors no longer require a PostgreSQL version."""
+    janitor = DatabaseJanitor(user="user", host="host", port="1234", dbname="database_name")
+    assert janitor.version is None
+
+
 @pytest.mark.parametrize("version", (VERSION, 10, "10"))
```

Ruff (0.16.1)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list`

Use `list` for parameter values

(PT007)